parm/fv3lam_rrfs.xml

```diff
@@ -1784,6 +1832,20 @@
        <scale>-3.0</scale>
        </param>
 
+       <param>
+       <shortname>GSD_MAX_UPHL_ON_SPEC_HGT_LVL_ABOVE_GRND_0-2km</shortname>
+       <pname>MXUPHL</pname>
```

Comment:
@AndrewBenjamin-NOAA This field was not generated in your test. You may remove it.